api/tests/unit/config/fixtures/data_fixtures/AppFixturesTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\unit\config\fixtures\data_fixtures;
+
+use App\DataFixtures\AppFixtures;
+use App\Entity\Question;
+use App\Entity\Quiz;
+use PHPUnit\Framework\TestCase;
+
+final class AppFixturesTest extends TestCase
+{
+
+    private $appFixtures;
+    private $objectManager;
+
+    protected function setUp(): void
+    {
+        $objectManager = $this->getMockBuilder('Doctrine\Persistence\ObjectManager')
```

Comment:
Please import `Doctrine\Persistence\ObjectManager` and use the following
`ObjectManager::class` instead of a string